tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -2492,3 +2492,900 @@ def max_call(x):
     f1 = epyccel(max_call, language = language)
     x = randint(99,size=10)
     assert(f1(x) == max_call(x))
+
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = [
+            pytest.mark.skip(reason="arrays not implemented"),
```

Comment:
This is no longer a valid reason. Could you be more specific

Reply:
i have discussed with Moad about that, and we found two options, since we haven’t implemented tuples in C, the `shape` function won't work, so we have either make the reason more specific i suggest "tuples not implemented yet" or do not work we `shape` function and do not test the length of the array, because we calculate it through the return of the `shape` function

Reply:
`tuples not implemented yet` seems fine to me